Makefile

```diff
@@ -35,6 +35,10 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	pip-compile --upgrade -o requirements/quality.txt requirements/quality.in
 	pip-compile --upgrade -o requirements/test.txt requirements/base.in requirements/test.in
 	pip-compile --upgrade -o requirements/travis.txt requirements/travis.in
+	cat requirements/quality-extra.in >> requirements/quality.txt
```

Comment:
This came straight from the cookiecutter, so would it make sense to also update it over there?

https://github.com/edx/cookiecutter-django-app/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/Makefile#L39